### PR TITLE
add v4 branch to plugin

### PR DESCRIPTION
**Pull request by mjudeikis**
add v4 ARO branch to pluging

### CODE REVIEW

File: cluster/ci/config/prow/plugins.yaml [hunk 1]
```diff
@@ -824,6 +824,8 @@ config_updater:
       name: job-config-misc
     ci-operator/jobs/**/*release-next*.yaml:
       name: job-config-misc
+    ci-operator/jobs/**/*openshfit-v*.yaml:
```
Comment (droslean) [inline cluster/ci/config/prow/plugins.yaml]: you have a typo here. :)

Comment (mjudeikis) [reply cluster/ci/config/prow/plugins.yaml]: fixed :D

**Pull request merged**